watertap/costing/watertap_costing_package.py

```diff
@@ -193,16 +193,8 @@ def build_global_params(self):
 
         # Define standard material flows and costs
         self.defined_flows["electricity"] = self.electricity_base_cost
-        self.defined_flows["NaOCl"] = (
-            74.44e-3
-            * (pyo.units.kg / pyo.units.mol)
-            * (self.naocl_cost / self.naocl_purity)
-        )
-        self.defined_flows["CaOH2"] = (
-            74.093e-3
-            * (pyo.units.kg / pyo.units.mol)
-            * (self.caoh2_cost / self.caoh2_purity)
-        )
+        self.defined_flows["NaOCl"] = self.naocl_cost / self.naocl_purity
+        self.defined_flows["CaOH2"] = self.caoh2_cost / self.caoh2_purity

```

Comment:
Why isn't the molecular weight needed anymore?

Reply:
Per @andrewlee94's suggestion, the molecular weight is part of the thermo configuration, so we use the conversion there in place of hardcoding it here.

Then the `flow` uses here should be in `kg/s` as opposed to `mol/s`, meaning the costing package doesn't also need to know the molecular weight.

Reply:
I think it makes sense.